### PR TITLE
Adding Dockerfile and docker-compose

### PR DESCRIPTION
**Pull request by patbaumgartner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+FROM python:3.9.6-slim-buster as base
+
+# Setup env
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONFAULTHANDLER 1
+ENV PATH=/home/bpauser/.local/bin:$PATH
```
Comment (brianleect) [inline Dockerfile:8]: Thanks for your PR Patrick, as I'm not particularly familiar with Docker and setting it up. I would like to clarify regarding the declaration of your PATH variable as it seems to be a local path specific to you? ('bpauser') bpauser seems to have been referenced in line 16,19,31,44 & 48 as well.

Would the "loading of the docker image" pose a problem for other potential users due to this PATH variable? What would be the expected steps to be taken for other users to utilize this Dockerfile?

The rest of the code looks good to me. 

Cheers Brian

**Review by brianleect — COMMENTED** [review]
Clarification on PATH variable declaration.

File: Dockerfile (hunk 1 above)
Comment (patbaumgartner) [reply Dockerfile:8]: You may have seen that in the Dockerfile I created a user called pbauser (binance-pump-alert-user) which is used to execute the app. It's very important that we do not run the script with the root user. This is considered best practice. Anyhow in the whole 3 Docker Images which are produced during the build, to keep the resulting image small, I have not added any specific things which are related to me. 

I understand that my name and the username in the image could give the impression, it's related to me. 👍

Comment (brianleect) [reply Dockerfile:8]: Thanks for the clarification! Everything looks good, will merge the pull.

**Pull request merged**